src/main/java/com/softserve/academy/spaced/repetition/controller/CourseRatingController.java

```diff
@@ -35,12 +34,12 @@
     }
 
     @PostMapping("/api/private/course/{courseId}")
-    public ResponseEntity addCourseRating(@RequestBody RatingDTO ratingDTO, @PathVariable Long courseId) throws RatingsBadValueException, NotAuthorisedUserException, UserStatusException {
+    public ResponseEntity addCourseRating(@RequestBody RatingDTO ratingDTO, @PathVariable Long courseId) throws NotAuthorisedUserException, UserStatusException {
         if ((ratingDTO.getRating() >= MIN_RATING) && (ratingDTO.getRating() <= MAX_RATING)) {
             courseRatingService.addCourseRating(ratingDTO.getRating(), courseId);
            return new ResponseEntity(HttpStatus.CREATED);
         } else {
-            throw new RatingsBadValueException();
+            throw new IllegalArgumentException("Rating can't be less than 1 and more than 5");
```

Comment:
And it appears that hte same check is used in three different places. Lets create a separate issue.